lib/features/greeting_screen/widgets/recetly_active_widget.dart

```diff
@@ -53,7 +56,7 @@ class RecentlyActiveGameWidget extends StatelessWidget {
       child: Container(
         decoration: BoxDecoration(
           color: context.colorTheme.greyLight,
-          borderRadius: BorderRadius.circular(7),
+          borderRadius: BorderRadius.circular(8),
```

Comment:
is there a reason why not `AppUiConfig.radius` here?

Reply:
I created another class because many containers on project has the same radious.

Reply:
Yea I will change it. I didn't notice